haystack/document_store/faiss.py

```diff
@@ -130,7 +131,8 @@ def update_embeddings(self, retriever: BaseRetriever, index: Optional[str] = Non
         doc_meta_to_update = []
         for vector_id, doc in enumerate(documents[i : i + self.index_buffer_size]):
```

Comment:
Just small suggestion
Better not to get `vector_id` from `enumerate`
And we can initialise `vector_id` in beginning as follows -
```
vector_id = self.faiss_index.ntotal
``` 
And then assign `vector_id` to `meta["vector_id"] = vector_id` only but later increment `vector_id` for example -
```
meta["vector_id"] = vector_id 
vector_id += 1
```
This way -
1) We can enable multiple `write_documents` calls
2) `update_embeddeings` function will not overwrite existing `faiss_index` data

Reply:
Good point! Makes sense